pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -549,8 +556,7 @@ func getLocalIP() ([]v1.NodeAddress, error) {
 					if ipnet.IP.To4() != nil {
 						// Filter external IP by MAC address OUIs from vCenter and from ESX
 						var addressType v1.NodeAddressType
```

Comment:
@frapposelli @andrewsykim should we fix this to? Its value is never set.

Reply:
I think we should but can be done in a separate PR, didn't want this one to go out of scope.